docs/02-developers/05-smart-contracts/04-verify-smart-contracts/Verify-Smart-Contracts-using-Blockscout/index.md

```diff
@@ -0,0 +1,181 @@
+
+   ![File](./images/choose.png)
+   - If you don’t have an account, create one by providing an email address or choose web3 wallet. Verify your email if prompted.
+
+4. **Access Your Account Settings**:
+   - Once logged in, click on your profile (often represented by your wallet address or a user icon) and select "Account" or "API Keys" from the dropdown menu.
+
+5. **Generate an API Key**:
+![File](./images/add.png)
+   - In the API Keys section, click "Create New API Key" or a similar button.
+
+   ![File](./images/name.png)
+   - Provide a name for the API key (e.g., "Foundry Verification") to help you identify its purpose.
+   - Copy the generated API key and store it securely. You will not be able to view it again after leaving the page.
+
+6. **Use the API Key in Foundry**:
+   - Add the API key to your `foundry.toml` configuration file under the `[etherscan]` section, as shown in the **Configuration** section below. Replace `"your api key"` with the key you obtained.
+
+:::tip[Tip]
+Keep your API key confidential and avoid sharing it publicly. If you suspect it has been compromised, regenerate a new key from your Blockscout account.
+:::
+
+---
+
+### Integration into the Tutorial
+
+To integrate this section, place it after the **Prerequisites** section and update the **Configuration** section to reference it. For example, modify the **Configuration** section to start with:
+
+---
+
+## Configuration
+
+Before proceeding, ensure you have obtained a Rootstock API key from Blockscout as described in the **Obtaining a Rootstock API Key from Blockscout** section. Create or update the `foundry.toml` file in your project root to include Rootstock network configurations. Add the following:
+
+```toml
+[profile.default]
+src = "src"
+out = "out"
+libs = ["lib"]
+solc_version = "0.8.24"
+optimizer = true
+optimizer_runs = 200
+evm_version = "london"
+
+[rpc_endpoints]
+rskTestnet = "${RSK_TESTNET_RPC_URL}"
+rskMainnet = "${RSK_MAINNET_RPC_URL}"
+anvil = "http://127.0.0.1:8545"
+
+[etherscan]
+rskTestnet = { key = "your api key", url = "https://rootstock-testnet.blockscout.com/api" }
+rskMainnet = { key = "your api key", url = "https://rootstock-blockscout.com/api" }
+```
+
+---
+
+### Notes
+- The steps assume Blockscout’s user interface remains consistent. If the interface changes, users may need to adapt to the new layout, but the general process (sign in, access account, generate API key) should remain similar.
+- The tutorial avoids hardcoding specific URLs or button names that might change, using generic terms like "API section" or "Account Settings" to ensure longevity.
+- The added section maintains the tutorial’s Markdown formatting and includes a tip box for best practices, aligning with the existing style.
+
+## Usage
+
+To verify a deployed contract, use the `forge verify-contract` command. You need the contract address and the contract name (as defined in your Solidity file). Run the following command:
```

Comment:
There's no flow of steps from using the foundry starter kit to compile the contract and use the forge contract command before verifying on the explorer